src/api/contributors/contributors.controller.js

```diff
@@ -5,7 +5,51 @@ module.exports = async function getContributors(req, res) {
   try {
     const response = await fetch(contribURL);
     const data = await response.json();
-    return res.json(data);
+    const finalResponse = {
+      data: data
+        .sort((a, b) => new Date(a.joined) - new Date(b.joined))
+        .map(({
+          name,
+          github: username,
+          image,
+          countryCode,
+          teamIds,
+          active,
+          joined,
+        }, key) => ({
+          type: 'contributor',
+          id: key,
```

Comment:
I don't think using the index in the array would work as key. When someone joins everyone after them will have a new id. What about using the GitHub username as unique identifier?

Reply:
Do we need the key to be consistent though?

Reply:
Although it will change per contributor the indices will always be unique. @sbibow

Reply:
They would be unique, but when we implement the `/contributor/{id}` route it would be a mess. It would still work but caching on that route will not work, since it might change whenever. The GitHub username is also guaranteed to be unique **and** it will be easy to query a specific user when we implement the next routes.